mono-traversable/src/Data/MonoTraversable.hs

```diff
@@ -1252,6 +1385,10 @@ instance MonoPointed (Tree a) where
 instance (Applicative f, Applicative g) => MonoPointed ((f :+: g) a) where
     opoint = R1 . pure
     {-# INLINE opoint #-}
+-- | @since ????????????
```

Comment:
there's a lot of commented code here; is this meant to be a draft PR?

Reply:
Thanks for the draft suggestion. Could you give you opinion on #229 that is related to this?